common/source/docs/common-buttons.rst

```diff
@@ -13,20 +13,22 @@ Up to four external buttons or switches connected to the autopilot can be config
 Basic Setup
 -----------
 
-For autopilots with an IOMCU (e.g. Pixhawk, CubeOrange) the external button or switch should be connected to the autopilot's AUX outputs (not the MAIN outputs).  To ease configuration and avoid timer conflicts it may be best to use AUX outputs at the high end (e.g. AUX5 or AUX6) far from outputs used for motors and servos.  For autopilots without an IOMCU (e.g. pixracer, etc) the main outputs may be used.
+In order to use a button, an autopilot pin must be configured as a GPIO. Many autopilots have dedicated pins for this, but some do not. In those cases, a normal SERVO/MOTOR output must be configured as a GPIO pin instead of an :ref:`common-rcoutput-mapping`.
+
+For autopilots with an IOMCU (e.g. Pixhawk, CubeOrange) the external button or switch should be connected to the autopilot's AUX outputs (not the MAIN outputs).  To ease configuration and avoid timer conflicts it may be best to use AUX outputs at the high end (e.g. AUX5 or AUX6) far from outputs used for motors and servos.  For autopilots without an IOMCU (e.g. pixracer, etc) the normal SERVO/MOTOR outputs may be used.
 
 - Set :ref:`BTN_ENABLE<BTN_ENABLE>` = 1 (Enable) and refresh parameters
-- If using 4.2 (or later) set :ref:`SERVOx_FUNCTION<SERVO14_FUNCTION>` = -1 (GPIO) where "x" is the servo output channel connected to the button (e.g. SERVO14_FUNCTION is for AUX output 6)
-- If using 4.1 (or earlier) on an autopilot with an IOMCU (e.g. Pixhawk, CubeOrange) set BRD_PWM_COUNT low enough so that the servo output may be used for reading an analog voltage (aka GPIO) or PWM input.  E.g. BRD_PWM_COUNT = 4 means only AUX outputs 5 and 6 can be used, BRD_PWM_COUNT = 0 means AUX outputs 1 to 6 can be used.
+- If using 4.2 (or later) set the ``SERVOx_FUNCTION`` = -1 (GPIO) where "x" is the servo output channel connected to the button (e.g. SERVO9_FUNCTION is for AUX output 1 for boards using an IOMCU)
```

Comment:
we have a little difference of opinion on how to handle these "x" parameters.  I like to link to one of the parameters anyway while you like to put them in quotes.  Not a big deal of course either way but I (of course) like my way better.

Reply:
the problem I have is the link arrives at a parameter that does not match the highlited text...alternatively we can explicitly show it as a stated example,and make them match...either way is fine with me...